### DISCUSSION TITLE
Zustand for Flutter?

### DISCUSSION THREAD

**Discussion by yahya-mohammed07** [pmndrs/zustand#2007] [Ideas]
Hii, any chance we will see Zustand in flutter?

**Comment by josiahsrc** [comment]
I created a library to bring zustand to flutter https://github.com/josiahsrc/flutter_zustand. It's still in early stages, but I think it has claws 😄 Hope you all enjoy!

@dai-shi  I hope this is well received. I tried to mirror Zustand's API as closely as possible. It would be awesome to get your feedback on it! More details [here](https://github.com/josiahsrc/flutter_zustand/blob/main/docs/motivation.md) on the motivation behind the project.

**Reply by dai-shi** [reply]
It's surprising that Zustand concept helps other frameworks / languages. But it's nice if it works.

**Reply by josiahsrc** [reply]
Flutter's state management systems today are very boilerplatey and/or require code generation. Zustand's API design solves those quite nicely imho, which was one of the reasons for me porting it over.  I think it's a good fit, but time will tell what the community thinks.
